msi/GitProduct.wxs

```diff
@@ -115,15 +124,15 @@
     <Feature Id='Git'>
       <Component Directory='INSTALLFOLDER'>
         <RegistryValue Root='HKLM' Key='SOFTWARE\GitForWindows' Name='InstallFolder' Value='[INSTALLFOLDER]' Type='string' />
-        <Environment Name='PATH' Value='[BinFolder]' Action='set' Part='last' System='yes' />
         <RemoveFolder Directory='GitProgramMenuFolder' On='uninstall' />
         <RemoveFolder Id="RemoveApplicationFolder" Directory='ApplicationFolder' On='uninstall' />
       </Component>
       <Component Directory="INSTALLFOLDER">
         <RegistryValue Root="HKLM" Key="Software\GitForWindows" Name="CurrentVersion" Type="string" Value="$(var.Version)" />
       </Component>
-      <Component Directory='INSTALLFOLDER'>
-        <RegistryValue Action='write' Type='string' Root='HKLM' Key='SOFTWARE\GitForWindows' Name='CliEnvironment' Value='[CLIENVIRONMENT]' />
+      <Component Directory="INSTALLFOLDER">
+        <RegistryValue Action='write' Type='string' Root='HKLM' Key='SOFTWARE\GitForWindows' Name='CliEnvironment' Value='[CLIENVIRONMENT]' KeyPath="yes" />
+        <Environment Name="PATH" Value="[CLIENVIRONMENTPATHS]" Action="set" Part="last" System="yes" />
```

Comment:
Now, I do not understand this. I thought that the user can choose to leave the `PATH` variable alone?